src/v2/view-builder/views/IdentifierView.js

```diff
@@ -42,7 +42,20 @@ const Body = BaseForm.extend({
     if (!hasUISchemas) {
       this.$el.find('.button-primary').hide();
     }
-  }
+  },
+  showMessages () {
+    // render messages with class INFO as warning callout
```

Comment:
nit:
maybe elaborate a little more like
- `as warning callout for identify view`
- this logic is intended for handling `unknow-user` case at identify view but the consequence is any `messages` along with `identify` view would render as callout regardless it's only meant for `unknown-user` case or not. don't have good answer now but let's put comments here.